ccmlib/common.py

```diff
@@ -39,6 +40,14 @@
 CONFIG_FILE = "config"
 CCM_CONFIG_DIR = "CCM_CONFIG_DIR"
 
+logger = logging.getLogger('ccm')
+ccm_log_level = os.environ.get('CCM_LOG_LEVEL', None)
+if ccm_log_level is not None:
+    ccm_log_level = ccm_log_level.upper()
+elif os.environ.get('PRINT_DEBUG', '').lower() in ('yes', 'true'):
```

Comment:
`PRINT_DEBUG`  here is pointless, and doesn't actually does what it suggest like in dtest, I think it should be removed from here.